third_party/katana/lib/usdKatana/readPoints.cpp

```diff
@@ -34,6 +34,49 @@ PXR_NAMESPACE_OPEN_SCOPE
 
 
 static FnKat::Attribute
+_GetPositionAttr(
+        const UsdGeomPoints& points,
+        const PxrUsdKatanaUsdInPrivateData& data)
+{
+    // Because of the logic used to gather these (in PxrUsdInOp::InitUsdInArgs),
+    // we can use the size of this vector as a simple test for whether motion
+    // blur is enabled, without needing access to the cook interface.
+    const std::vector<double>& motionSampleTimes =
+        data.GetUsdInArgs()->GetMotionSampleTimes();
```

Comment:
@spiffmon Done! Both using the points attr in the query and all the motion samples, not just the shutter open and close.